<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoiceTestWdg</class>
 <widget class="QWidget" name="VoiceTestWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VoiceTestWdg</string>
  </property>
  <widget class="QLabel" name="backLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>380</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>智慧云视频语音测试向导</string>
   </property>
  </widget>
  <widget class="QLabel" name="downLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>329</y>
     <width>450</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="contentWidget" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>39</y>
     <width>442</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>10</y>
     <width>17</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nextBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
